ddsrecorder/src/cpp/tool/DdsRecorder.cpp

```diff
@@ -29,21 +31,18 @@ using namespace eprosima::ddsrecorder::participants;
 using namespace eprosima::utils;
 
 DdsRecorder::DdsRecorder(
-        const yaml::RecorderConfiguration& configuration,
+        yaml::RecorderConfiguration& configuration,
```

Comment:
I would copy it to an attribute and modify that one, it could also be helpful for debugging.